parla/task_runtime.py

```diff
@@ -1755,19 +1798,33 @@ def update_launched_task_count(self, task, dev, counts):
         else:
             self._device_launched_datamove_task_counts[dev] += counts
 
-    def update_mapped_task_count(self, task, dev, counts):
-        with self._monitor:
+    def get_launched_compute_task_count(self, dev):
+        with self._launched_count_monitor[dev]:
+            return self._device_launched_compute_task_counts[dev]
+
+    def get_launched_datamove_task_count(self, dev):
+        with self._launced_count_monitor[dev]:
```

Comment:
it should be `_launched_count_..` not launced

Reply:
Thanks!. Thats what I get for only testing with a manual test case.